util/dvsim/doc/testplanner.md

```diff
@@ -40,14 +40,36 @@ The following attributes are used to define each testpoint, at minimum:
 
     Full [Markdown]({{< relref "doc/rm/markdown_usage_style" >}}) syntax is supported when writing the description.
 
-* **tests: list of written test(s) for a testpoint**
+* **tests: list of written test(s) for this testpoint**
 
     The testplan is written in the initial work stage of the verification [life-cycle]({{< relref "doc/project/development_stages#hardware-verification-stages" >}}).
     Later, when the DV engineer writes the tests, they may not map one-to-one to a testpoint - it may be possible that a written test satisfactorilly addresses multiple testpoints; OR it may also be possible that a testpoint needs to be split into multiple smaller tests.
     To cater to these needs, we provide the ability to set a list of written tests for each testpoint.
     It is used to not only indicate the current progress so far into each milestone, but also map the simulation results to the testpoints to generate the final report table.
     This list is initially empty - it is gradually updated as tests are written.
 
+* **tags: list of tags relevant for this testpoint**
+
+    Tags represent the need to run the same testpoint under specific conditions, in additional platforms or with specific set of build / runtime options.
+    At the moment, tags are not strictly defined - users are free to come up with their own set of tags.
+    The follow examples of tags illustrate the usage:
+
+```hjson
+  // Run this testpoint on verilator and fpga as well.
+  tags: ["verilator", "fpga_cw310"]
+
+  // Run this testpoint in gate level and with poweraware.
+  tags: ["gls", "pa"]
+
+  // Run this testpoint with mask ROM (will use test ROM by default).
+  tags: ["mask_rom"]
+
+  // Run this testpoint as a post-Si test vector on the tester.
+  tags: ["vector"]
+```
+
+    The testplan from the documentation point of view, can be filtered by tag, so that generated testplan table is restricted to that aspect.
```

Comment:
nit: "so that generated testplan" --> "so that the generated testplan"

Reply:
Fixed.